common/changes/@rushstack/heft-config-file/config-file-null_2025-04-16-00-37.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-config-file",
+      "comment": "Allow use of the value `null` to discard any value set for the property from a parent config file..",
+      "type": "minor"
```

Comment:
Please make a PR to update these docs:

https://heft.rushstack.io/pages/advanced/heft-config-file/

Reply:
https://github.com/microsoft/rushstack-websites/pull/278